src/config/deck-layers.ts

```diff
@@ -141,6 +141,21 @@ export function labelsLayer(isRetina: boolean) {
   );
 }
 
+export function boundariesLayer(level: 'parish' | 'community') {
+  return new MVTLayer(
+    {
+      id: `boundaries-${level}`,
+      data: `/vector/data/boundaries_${level}.json`,
+      binary: true,
+      filled: false,
+      refinementStrategy: 'best-available',
+      getLineWidth: level === 'parish' ? 2 : 1,
+      lineWidthUnits: 'pixels',
+    } as any,
+    border([190, 190, 190, 255]),
```

Comment:
I needed to add `as any` here for this to compile.
```suggestion
    border([190, 190, 190, 255]) as any
```

Reply:
Interesting, I think this might be to do with the external deck.gl typings disappearing on every new installed dependency, which meant that the types weren't properly being checked when I was developing this. The relevant issue is here: https://github.com/danmarshall/deckgl-typings/issues/133
I'm going to add the `postinstall` step suggested in that issue to avoid this in the future.